spacewalk/certs-tools/rhn_bootstrap_strings.py

```diff
@@ -441,7 +441,7 @@ def getRegistrationStackSh():
         elif [ -f /etc/redhat-release ]; then
             grep -v '^#' /etc/redhat-release | grep -q '\(Red Hat\)' && BASE="res"
             VERSION=`grep -v '^#' /etc/redhat-release | grep -Po '(?<=release )\d+'`
-	elif [ -f /etc/openEuler-release ]; then
+	    elif [ -f /etc/openEuler-release ]; then
```

Comment:
Looks to me this belongs to another PR?

Reply:
Restored. No idea why I was seeing a wrong tabulation... At any case, you're right, this is not the place to fix it.